File: CommonDomain/Show.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommonDomain
{
    [Serializable]
    public class Show : Entity<int>
    {
        private string _artistName;
        private string _showName;
        private string _address;
        private DateTime _time;
        private int _availableSeats;
        private int _soldSeats;

        public Show(string aristName, string showName, string address, DateTime time, int availableSeats, int soldSeats)
        {
            this._artistName = aristName;
            this._showName = showName;
            this._address = address;
            this._time = time;
            this._availableSeats = availableSeats;
            this._soldSeats = soldSeats;
        }

        public string GetArtistName()
        {
            return _artistName;
        }

        public void SetArtistName(string artistName)
        {
            this._artistName = artistName;
        }

        public string GetShowName()
        {
            return _showName;
        }

        public void SetShowName(string showName)
        {
            this._showName = showName;
        }

        public string GetAddress()
        {
            return _address;
        }

        public void SetAddress(string address)
        {
            this._address = address;
        }

        public DateTime GetStartTime()
        {
            return _time;
        }

        public void SetStartTime(DateTime startTime)
        {
            this._time = startTime;
        }

        public int GetAvailableSeats()
        {
            return _availableSeats;
        }

        public void SetAvailableSeats(int availableSeats)
        {
            this._availableSeats = availableSeats;
        }

        public int GetSoldSeats()
        {
            return _soldSeats;
        }

        public void SetSoldSeats(int soldSeats)
        {
            this._soldSeats = soldSeats;
        }
    }
}

File: CommonDomain/src/main/java/domain/Show.java
package domain;

import java.time.LocalDateTime;
import java.util.Objects;

public class Show extends Entity<Integer> {
    String artistName;
    String showName;
    String address;
    LocalDateTime startTime;
    Integer availableSeats;
    Integer soldSeats;

    public Show(String artistName, String showName, String address,
                LocalDateTime startTime, Integer availableSeats,
                Integer soldSeats) {
        this.artistName = artistName;
        this.showName = showName;
        this.address = address;
        this.startTime = startTime;
        this.availableSeats = availableSeats;
        this.soldSeats = soldSeats;
    }

    public String getArtistName() {
        return artistName;
    }

    public void setArtistName(String artistName) {
        this.artistName = artistName;
    }

    public String getShowName() {
        return showName;
    }

    public void setShowName(String showName) {
        this.showName = showName;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public LocalDateTime getStartTime() {
        return startTime;
    }

    public void setStartTime(LocalDateTime startTime) {
        this.startTime = startTime;
    }

    public Integer getAvailableSeats() {
        return availableSeats;
    }

    public void setAvailableSeats(Integer availableSeats) {
        this.availableSeats = availableSeats;
    }

    public Integer getSoldSeats() {
        return soldSeats;
    }

    public void setSoldSeats(Integer soldSeats) {
        this.soldSeats = soldSeats;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        Show show = (Show) o;
        return Objects.equals(artistName, show.artistName) && Objects.equals(showName, show.showName) &&
                Objects.equals(address, show.address) && Objects.equals(startTime, show.startTime) &&
                Objects.equals(availableSeats, show.availableSeats) &&
                Objects.equals(soldSeats, show.soldSeats);
    }

}